### PR TITLE
Cube preview pagination

### LINKED ISSUE #1049: Cube-Designer: Improve Pagination
Show total amount of lines in Cube-Designer, show final amount of pages.

### PR DESCRIPTION
**Pull request by martinmaillard**
Closes #1049

### CODE REVIEW

**Comment by martinmaillard** [comment]
I just had to run the tests enough times I guess....

File: ui/src/components/CubePreview.vue [hunk 1]
```diff
@@ -84,34 +84,41 @@
...
                 <b-tooltip label="Next page">
                   <b-button
                     icon-left="chevron-right"
                     @click="page = page + 1"
-                    :disabled="!observations.data || observations.data.length === 0"
+                    :disabled="!hasNextPage"
                   />
                 </b-tooltip>
+                <span class="ml-4">Page</span>
+                <b-input v-model.number="page" type="number" class="is-inline-block w-20" />
```
Comment (tpluscode) [inline ui/src/components/CubePreview.vue:104]: The binding causes immediate reload on typing. It should be deferred or happen on `blur`. Right now if I type, say, 6800, there will be requests or pages `6`, `68`, `680` and `6800` :)

**Review by tpluscode — CHANGES_REQUESTED** [review]

File: ui/src/components/CubePreview.vue (hunk 1 above)
Comment (martinmaillard) [reply ui/src/components/CubePreview.vue:104]: Good catch

**Review by tpluscode — APPROVED** [review]
looks good

**Pull request merged**